model-optimizer/mo/main.py

```diff
@@ -205,6 +205,12 @@ def raise_ie_not_found():
     except Exception as e:
         raise_ie_not_found()
 
+    if 'data_type' in argv and argv.data_type in ['FP16', 'half']:
+        argv.data_type = 'FP32'
+        argv.compress_fp16 = True
+    else:
+        argv.compress_fp16 = False
```

Comment:
There is no default value for `argv.compress_fp16` which leads to 
```
[ ERROR ]  'Namespace' object has no attribute 'compress_fp16'
```

Reply:
I prepared fix: #8545

Reply:
Thanks!